.github/workflows/dummy.yml

```diff
@@ -0,0 +1,30 @@
+name: Add a Dummy commit to trigger build
+
+on:
+  push:
+    branches: master
+    paths-ignore:
+      - .github/workflows/dummy.yml
+  schedule:
```

Comment:
This workflow will be executed upon every merge into master. Would that re-schedule the dummy commits? What is the effect? Will we be scheduling many redundantly scheduled ones?

Reply:
this just ignores push on workflow def file

Reply:
So, would it reschedule a new cron every time?

Reply:
only on schedule, removed the push path as was having conflicts